api_spec/edge_api.json

```diff
@@ -178,6 +253,31 @@
                         "type": "string"
                     }
                 }
+            },
+            "BulkIdentitiesData": {
+                "required": true,
+                "type": "object",
+                "properties": {
+                    "data": {
+                        "type": "array",
+                        "properties": {
+                            "items": {
```

Comment:
@EdsnLoor this still is not correct. We're now defining an `items` key in the request body which is not correct. Can you make sure you are running the docs site locally and verifying that they look correct before resolving the conversations please?